smdebug/trials/trial_catalog.py

```diff
@@ -34,7 +34,7 @@ def __init__(self,endpoint,port):
         self.endpoint = endpoint
         self.port = port
         self.client = InfluxDBClient(host=self.endpoint, port=self.port)
-        self.client.switch_database('tornasole_deb')
+        self.client.switch_database('deb')
```

Comment:
is this class needed? not in this PR but subsequent PR we should remove this class. can you track it in github issue